backend/src/fun_forecast_backend/weather/dataclasses.py

```diff
@@ -56,3 +56,11 @@ class WeatherDataProperties:
 @dataclass
 class WeatherData:
     properties: WeatherDataProperties
+
+
+@dataclass
+class LatLongResponse:
+    lat: float
+    long: float
+    name: str
+    full_name: str
```

Comment:
Let's make this camelCase and let the json encoder do the underscoring
```suggestion
    fullName: str
```

Reply:
Point against that suggestion - python standards dictate underscores for variables and attributes

Reply:
Oh then we should make them all underscores

Reply:
I see the others are all camel. I care more about consistency than anything so if this small change makes it all consistent, all g from me